README.md

```diff
@@ -107,4 +107,4 @@ or via `conda`:
 conda install -c conda-forge zarr
 ```
 
-For more details, including how to install from source, see the [installation documentation](https://zarr.readthedocs.io/en/stable/#installation).
+For more details, including how to install from source, see the [installation documentation](https://zarr.readthedocs.io/en/stable/installation.html).
```

Comment:
Tried this link and it seems to not work for me. This appears to though. Sorry for not catching this earlier.

```suggestion
For more details, including how to install from source, see the [installation documentation](https://zarr.readthedocs.io/en/stable/index.html#installation).
```

Reply:
You're right. 
I think this update will be best when the changes in 'latest' have been brought into "stable"
@jakirkham will it be alright if I close the PR and reopen when changes in 'latest' have been brought into "stable" or i keep it open?